selenium_tests/CanonizerAccountSettingsPage.py

```diff
@@ -1,5 +1,7 @@
 from CanonizerBase import Page
-from Identifiers import AccountSettingsIdentifiers, LogoutIdentifiers, AccountSettingsChangePasswordIdentifiers, AddAndManageNickNamesIdentifiers, AccountSettingsManageProfileInfoIdentifiers, MySupportsPageIdentifiers, HomePageIdentifiers
+from Identifiers import AccountSettingsIdentifiers, LogoutIdentifiers, AccountSettingsChangePasswordIdentifiers, \
```

Comment:
Not sure, why it is changed to multiple lines, if there aren't any new classes to import.